code/TransferBlobs/Program.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Sas;
+
+namespace TransferBlobs
+{
+    class Program
+    {
+        static void Main(string[] args)
```

Comment:
Not sure how much it matters here, but do we want to use `async` main (C# 7.1+) so we can properly await `MoveMatchingBlobsAsync` on line 39?

Reply:
Yeah, makes sense. I've updated it.